tests/integration-tests/src/scenarios/setupNewChain.ts

```diff
@@ -15,6 +16,10 @@ scenario('Setup new chain', async ({ job }) => {
     job('initialize distribution system', initDistribution(defaultDistributionConfig)).requires(updateWorkerAccounts)
   }
 
+  if (!process.env.SKIP_FAUCET_INIT) {
+    job('Initialize Faucet', initFaucet).requires(leads)
```

Comment:
```suggestion
    job('Initialize Faucet', initFaucet).requires(updateWorkerAccounts)
```
Should be safer in case lead role account changes midway through `initFaucet` flow.